src/containers/AdminCampaignEdit.jsx

```diff
@@ -539,9 +588,10 @@ const mapQueriesToProps = ({ ownProps }) => ({
     variables: {
       campaignId: ownProps.params.campaignId
     },
-    pollInterval: 10000
+    pollInterval: 60000
   },
   organizationData: {
+    // TOOD: optOuts list on client side doesn't scale
```

Comment:
Is this in a GitHub issue?

Reply:
good idea!  I made one: https://github.com/MoveOnOrg/Spoke/issues/249